### ISSUE TITLE
Is it possible to duplicate a line

### ISSUE THREAD

**Issue by gauthierf** [mickaelr/jquery-timelineMe#19]
Hello, I'm just trying to make a timeline which i can use with one entry but with multiple outpout, so, i must du much than one path, beginning with one.

i want to do something like that :

```
    event 4.1  | | event 4.2
    event 3.1  | | event 3.2
                V
                | event 2
                | event 1
```

Does your librairy can handle something like this ?

I don't know if you really understood what I mean but it's hard to explain

**Comment by mickaelr** [comment]
Hi!

Correct me if I'm wrong but you're trying to created something that looks like git branches:
![capture_stepup1_5_6](https://cloud.githubusercontent.com/assets/708439/19079866/621edea6-8a55-11e6-90ce-e67505f8c364.png)
?

If it's the case, it's not possible yet, and I'll have to think a bit further on a way to implement that...

**Comment by gauthierf** [comment]
Yeah, I want to do something like that, I'll follow all the nexts upgrades to see how you implement it and in order to use it for my web interface
Thanks for your quick answer :D
